hax/hax/handler.py

```diff
@@ -50,10 +51,31 @@ def __init__(self, q: Queue, motr: Motr):
         self.is_stopped = False
         self.consul = ConsulUtil()
         self.eq_publisher = EQPublisher()
+        self.statuses: List = []
 
     def stop(self) -> None:
         self.is_stopped = True
 
+    def _update_process_status(self, event: ConfHaProcess) -> None:
```

Comment:
[Optional]
This version looks shorter and more readable:
```python
    @repeat_if_fails(wait_seconds=1)
    def _update_process_status(self, event: ConfHaProcess) -> None:
        # If a consul-related exception appears, it will
        # be processed by repeat_if_fails.
        #
        # This thread will become blocked until that
        # intermittent error gets resolved.
        self.consul.update_process_status(event)
```

Reply:
Agree.